lib/generators/mountain_view/component_generator.rb

```diff
@@ -1,8 +1,8 @@
-require "rails/generators/base"
+require_relative './mountain_view_generator_base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.